src/api/spec/models/workflow/step/link_package_step_spec.rb

```diff
@@ -9,24 +9,6 @@
                         token: token)
   end
 
-  RSpec.shared_context 'failed without link permissions' do
```

Comment:
I'm concerned about removing this shared context and the lines that use it. I'm not aware we check that an error is raised for disabled access packages in some other place of the test suite.

Do you think we are safe removing it?

Reply:
This is "just" `Package.get_by_project_and_name` -> `Package.check_source_access?`. If this is broken we have other problems than `LinkPackageStep`...